### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by Pontus-Wikstrom**
Added a header and changed some className names in order to let header and donationbox work independently.

### CODE REVIEW

File: Frontend/DAT257_Website/src/Components/header.tsx
Comment (David-Pixeln) [inline Frontend/DAT257_Website/src/Components/header.tsx:1]: This file seems useless, remove it before merging!

File: Frontend/DAT257_Website/src/Components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.Header {
```
Comment (David-Pixeln) [inline Frontend/DAT257_Website/src/Components/Header/Header.css:1]: lowercase h

**Review by David-Pixeln — CHANGES_REQUESTED** [review]
Fix what is commented!

File: Frontend/DAT257_Website/src/Components/Header/Header.css (hunk 1 above)
Comment (Pontus-Wikstrom) [reply Frontend/DAT257_Website/src/Components/Header/Header.css:1]: The Header className is for convention named Header, I can't use lwoer case h if I want to access it. Same goes for .DonationBox

File: Frontend/DAT257_Website/src/Components/header.tsx
Comment (Pontus-Wikstrom) [reply Frontend/DAT257_Website/src/Components/header.tsx:1]: removed

**Comment by David-Pixeln** [comment]
You also now have merge conflicts that should be rectified before merging!

**Review by David-Pixeln — APPROVED** [review]
Looks good!

**Pull request merged**